src/vivarium_census_prl_synth_pop/tools/cli_utils.py

```diff
@@ -0,0 +1,83 @@
+import re
+import shutil
+from datetime import datetime
+from pathlib import Path
+from typing import Optional, Tuple
+
+from loguru import logger
+
+from vivarium_census_prl_synth_pop.constants import paths
+from vivarium_census_prl_synth_pop.utilities import build_output_dir
+
+
+def validate_args(
+    mark_best: bool, test_run: bool, label_version: Optional[str] = None
+) -> None:
+    if mark_best and test_run:
+        raise RuntimeError(
+            "A test run can't be marked best. "
+            "Please remove either the mark best or the test run flag."
+        )
+    if label_version is not None:
+        expected_version_format = re.compile("^\d+\.\d+\.\d+$")
```

Comment:
Bugfix here: the original regex was allowing for non-digit parts (foo.bar.baz) and also was not limiting it to only three (1.2.3.4.5.6)